ai/core/src/unitycatalog/ai/core/utils/function_processing_utils.py

```diff
@@ -282,3 +282,29 @@ def supported_function_info_types():
         pass
 
     return types
+
+
+def sanitize_string_inputs_of_function_params(function_params: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Sanitize string inputs of function parameters to prevent injection attacks.
+
+    Args:
+        function_params: A dictionary of function parameters.
+
+    Returns:
+        A sanitized dictionary of function parameters.
+    """
+    sanitized_params = {}
+    for key, value in function_params.items():
+        if isinstance(value, str):
+            # Escape single quotes, backslashes, and control characters that would otherwise break Python code execution
```

Comment:
wondering if we could just use `"""{value}"""`

Reply:
This would convert
```
"what's databricks?" --> "what''s Databricks?" --> "'what''s Databricks?'" in the SQL command
```

Reply:
I just ran through testing that and anything that has a reserved character in generated markdown rendering code like a \n or a \t for formatting of generated code with long single lines will throw syntax errors or invalid line continuation errors for logical execution blocks. 
That being said, adding external triple quote wraps around the entire arg for a string provides another safeguard against some edge case syntax errors without requiring us to use something more intrusive :)